packages/terra-tabs/src/common-tabs/CommonTabContentStatusOverlay.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import ThemeContext from 'terra-theme-context';
+import StatusLayout from './StatusLayout';
+
+import WorkspaceContentStatusOverlayButton from './CommonTabContentStatusOverlayButton';
```

Comment:
can we rename this variable to be more meaningfull in terra-tabs context ?

Reply:
Renamed in https://github.com/cerner/terra-framework/pull/1693/commits/fe666a309668b324ca27e8d32550eca0a135d2d2